File: Rest-Api-V3/market-data-public/markets/markets.py
import requests

def fetch_market_data():
    market_code = 'BTC-USD-SWAP-LIN'
    url = f'https://api.ox.fun/v3/markets?marketCode={market_code}'

    try:
        response = requests.get(url)
        response.raise_for_status()  # Raise an error for bad responses
        print('Market Data:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching market data:', error)

fetch_market_data()

File: Rest-Api-V3/market-data-public/markets/markets.js
const axios = require('axios');

async function fetchMarketData() {
  const marketCode = 'BTC-USD-SWAP-LIN';
  const url = `https://api.ox.fun/v3/markets?marketCode=${marketCode}`;

  try {
    const response = await axios.get(url);
    console.log('Market Data:', response.data);
  } catch (error) {
    console.error('Error fetching market data:', error.response ? error.response.data : error.message);
  }
}

fetchMarketData();